backend/ordd_api/models.py

```diff
@@ -346,11 +349,29 @@ def gem_score_calculate(cls, inst):
 
         return score, score_th_norm
 
+    @classmethod
+    def gem_score_calculate_new(cls, inst):
+        if not inst.is_existing:
+            cat = 3
+        elif not inst.is_pub_available:
+            cat = 3
+        elif (inst.is_digital_form and inst.is_avail_online and
+              inst.is_avail_online_meta and inst.is_bulk_avail and
+              inst.is_machine_read and inst.is_avail_for_free and
+              inst.is_open_licence):
+            cat = 1
+        else:
+            cat = 2
+
+        return cat
+
     def save(self, *args, **kwargs):
         # to calculate scores, using m2m relationships, save a first
         # time the dataset before calculations is required
         super().save(*args, **kwargs)
         self.score, self.score_th_norm = self.gem_score_calculate(self)
+        self.score_new = 0
```

Comment:
Please add a small comment about the fact that this is hardcoded to zero for now.